### PR TITLE
Added slider component

### LINKED ISSUE #1827: Extract `Slider` from `antd`
Slider should be extracted from `antd`, into neetoUI, since additional styling is applied for colours, etc. wherever slider is used.

### PR DESCRIPTION
**Pull request by neenu-chacko**
- Fixes #1827 

**Description**

**Checklist**

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by neenu-chacko** [comment]
@ajmaln _a Please review.
@praveen-murali-ind  Please look into whether we need any stylings to be added for the slider.

File: src/components/formik/Slider.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { useField } from "formik";
+
+import Slider from "components/Slider";
+
+const FormikSlider = forwardRef(({ name, ...otherProps }) => {
+  const [field, meta, { setValue }] = useField(name);
+
+  return (
+    <Slider
+      error={meta.error || ""}
```
Comment (ajmaln) [inline src/components/formik/Slider.jsx:12]: Does the error get displayed in the UI?

File: tests/formik/Slider.test.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  it("renders without errors", () => {
+    const onSubmit = jest.fn();
+    const { container } = render(
+      <Formik initialValues={{ slider: 0 }} onSubmit={onSubmit}>
+        <Form>
+          <Slider name="slider" />
+        </Form>
+      </Formik>
+    );
+    expect(container).toBeInTheDocument();
```
Comment (ajmaln) [inline tests/formik/Slider.test.jsx:18]: Let's add a testid and query with it instead.

**Review by ajmaln — COMMENTED** [review]
@neenu-chacko _a Please check

**Comment by goutham-subramanyam** [comment]
@praveen-murali-ind I have made the changes. Please check.

cc: @ajmaln

**Review by praveen-murali-ind — APPROVED** [review]
LGTM

**Review by ajmaln — APPROVED** [review]

**Pull request merged**